src/Microsoft.ML.Ensemble/Trainer/Multiclass/MulticlassDataPartitionEnsembleTrainer.cs

```diff
@@ -64,8 +64,8 @@ public Arguments()
                             // non-default column names. Unfortuantely no method of resolving this temporary strikes me as being any
                             // less laborious than the proper fix, which is that this "meta" component should itself be a trainer
                             // estimator, as opposed to a regular trainer.
-                            var trainerEstimator = new MulticlassLogisticRegression(env, LabelColumnName, FeatureColumnName);
-                            return TrainerUtils.MapTrainerEstimatorToTrainer<MulticlassLogisticRegression,
+                            var trainerEstimator = new LogisticRegressionMulticlassClassificationTrainer(env, LabelColumnName, FeatureColumnName);
```

Comment:
>LogisticRegressionMulticlassClassificationTrainer [](start = 55, length = 49)

this will get ppl confused because it has both Regression and Multiclass on the name, but can't think of a good way to deal with it. would it be ok to just call it Logit @wschin @TomFinley

Reply:
I would especially hesitate to call it Logit . `Logits` has a different interpretation  related to the  unnormalized log-probabilities

---
In reply to: [264516270](https://github.com/dotnet/machinelearning/pull/2903#discussion_r264516270) [](ancestors = 264516270)